.github/workflows/schema.yml

```diff
@@ -13,7 +13,10 @@
 # limitations under the License.
 
 name: "Lint check"
-on: [push]
+on:
```

Comment:
Let's also add a concurrency group to cancel any in progress runs if new commit is pushed
```
concurrency:
  group: '${{ github.workflow }}-${{ github.head_ref || github.ref }}'
  cancel-in-progress: true
```

Reply:
Good Idea!